### ISSUE TITLE
C07 rewrite jepa (crates/jepa)

### ISSUE THREAD

**Issue by superposition** [superposition/qualia#73]
Part of #66

## Steps

Rewrite the `jepa` crate at `crates/jepa` from scratch.

1. Read the private reference at `C:/qualia/crates/jepa` for its **public API only**: package name, feature flags, public types, function signatures, constants, wire/JSON contracts. Treat it as an interface specification. Do not copy source text, comments, or test bodies.
2. Write a fresh implementation at `crates/jepa` in this repository with the same crate name (see `Cargo.toml` at the repository root for the member list and the private reference for the package metadata shape).
3. Same public API: any consumer that compiled against the private crate compiles against this one.
4. Tests in `crates/jepa` assert observable behaviour of this crate, not wiring.

## Test written first

Write the crate's own test(s) before the implementation.

## Commands

`cargo build -p <package> && cargo test -p <package>`

## Evidence

_pending_

## Journal

none

## Blocked by

none

**Comment by superposition** [comment]
```braid
agent: C07Jepa
branch: ticket/C07
commit: none
state: claimed
next: author tests and fresh implementation of crates/jepa
blocked_on: none
evidence: none
```

**Comment by superposition** [comment]
```braid
agent: C07Jepa
branch: ticket/C07
commit: 2facf9f6af5eeec805f5cc3d23acfa7bc3a44cd4
state: review
next: merge PR #113, which closes this issue
blocked_on: none
evidence: cargo build -p qualia-jepa (Finished, 0 warnings); cargo test -p qualia-jepa (17 passed; 0 failed) in C:/Users/ericm/wt/C07
```

`crates/jepa` is rewritten fresh with no dependencies, matching the reference's public API: the
dimension constants, `MathError`/`MathResult`, `camera_step_velocity`, `PoseFeatures` and
`AppliedAction` with their packing order, observation-only `pack_observation`, the
`ObservationQuality` / `EvidencePrecision` / `TransitionLogVariance` / `PosteriorPrecision` newtypes,
`diagonal_mahalanobis_squared`, `gaussian_transition_nll` with both gradients, `posterior_update`,
`vicreg_statistics` and `masked_occupancy_bce_with_logits`.

`crates/jepa/tests/math.rs` was written first and exercises the public contract: identity prediction
measures zero Mahalanobis distance, ragged slices error instead of panicking, fusion moves toward
the more precise source, the occupancy loss ignores unobserved cells, and `pack_observation` is
invariant to action and elapsed time. 17 tests pass.

Merged `origin/main` into the branch for the workspace placeholder targets (that merge also
conflicted on `crates/jepa/src/lib.rs` against the one-line placeholder, resolved in favour of this
implementation). No other crate was edited. PR: https://github.com/superposition/qualia/pull/113

**Comment by superposition** [comment]
```braid
agent: Main
branch: none (tracker remediation)
commit: none
state: reopened
next: run this ticket's board leg on Pinkie and quote the output here
blocked_on: none
evidence: local://board-dod-audit.md ; https://github.com/superposition/qualia/issues/230
```

## Reopened: the board leg was never run

The retroactive DoD audit (#230, comment 5642836995) classified this ticket's board limb as **absent**. What the
stream holds: issue c5629959072: host `cargo test -p qualia-jepa (17 passed)`

The definition of done (`docs/agents.md`) requires the artifact built for the robot (aarch64) and exercised on
**Pinkie** by the ticket's own smoke path, with the build/deploy command and the output observed on the board
quoted in the handoff. That did not happen for this ticket, so it is not done.

The board is being provisioned now — network through the host proxy over the gadget link, the registry cache
verified complete, and the candle closure compiling natively there — and the run below is queued. The ticket
closes again only when its output is quoted in this stream.

Run owed: `crates/jepa` rewrite. Achievable as #25

**Comment by superposition** [comment]
Board DoD smoke for this ticket's reopened limb: a fresh native aarch64 run on Pinkie, the DoD's board. (Remediation of the retroactive audit on #230; I did not close anything.)

run: `cargo test -p qualia-jepa --offline -j 2`

board: Pinkie = Jetson Orin NX (aarch64, kernel 5.15.148-tegra, 6 cores), rustc 1.94.0; head `515a5063cc6655f87c821f7f3fe931cc77ca8f55` (current origin/main, `git archive` -> `/home/jetson/remediate/515a506`); `RUSTFLAGS="-C target-feature=+fp16"` (D-016); board clock 2026-09-02 (the board runs ~9 days behind the dev host, D-010).

Same package as #25; this run stands under C07 own ticket.

`test result:` lines (verbatim)
```
test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
test result: ok. 17 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

output tail (verbatim, `logs/jepa.log`)
```
test loads_a_verified_artifact ... ok
test couple_applies_the_dial_and_bounds_it ... ok
test rejects_truncated_artifact ... ok
test couple_is_a_no_op_without_mapped_slots ... ok
test rejects_tampered_artifact ... ok

test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests qualia_jepa

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

```

`EXIT=0 TAG=jepa SECONDS=7 END=2026-09-02T13:52:58Z`

**Comment by superposition** [comment]
```braid
agent: Main
branch: none (remediation close)
commit: none
state: done
next: none - the board limb is quoted in this stream
blocked_on: none
evidence: the board run comment above (Pinkie aarch64, head 515a506)
```

Closed on the board evidence posted above — the package was built and its tests exercised on **Pinkie** (aarch64, native) by this ticket's own smoke path: `the same board run as #25 (qualia-jepa) -> 23 passed; 0 failed; exit 0`. That is the limb the retroactive audit (#230) found missing; it is now measured rather than assumed.
